draft-eggert-bcp45bis.md

```diff
@@ -170,12 +170,27 @@ Inappropriate postings include:
 
 # Security Considerations
 
-The IETF Chair, the IETF Executive Director, or a sergeant-at-arms {{SAA-SOP}}
-appointed by the Chair is empowered to restrict posting by a person, or of a
-thread, when the content is inappropriate and represents a pattern of abuse
-{{SAA-UPC}}. They are encouraged to take into account the overall nature of the
-postings by an individual and whether particular postings are an aberration or
-typical. Complaints regarding their decisions should be referred to the IAB.
+A sergeant-at-arms (SAA) appointed by the IETF Chair is empowered to
+restrict posting by a person, or of a thread, when the content is
+inappropriate and represents a pattern of abuse. They are encouraged to
+take into account the overall nature of the postings by an individual
+and whether particular postings are an aberration or typical.
+
+The sergeants-at-arms are intended to establish a self-moderation
+function on the community, by the community. The IETF Chair therefore
+SHOULD NOT appoint an SAA who is serving in a NomCom-appointed IETF
+leadership position. If an SAA is selected for such a position, they
+SHOULD consider stepping down as SAA.
+
+Apart from appointing SAAs, the IETF Chair SHOULD divorce themselves
+from the operation and management of the SAA team. This has been
+in practice for a while, and the SAA team has
+independently maintained definitions of abuse patterns {{SAA-UPC}} and
+operating procedures {{SAA-SOP}} for them. 
+
+Because an SAA serves at the discretion of the IETF Chair - even if the
+IETF Chair is not otherwise involved in the operation of the SAA team -
+complaints regarding any SAA decisions should be appealed to the IAB.
```

Comment:
```suggestion
SAA decisions should be appealed to the IAB.
```

Reply:
Read literally, that's encouraging all complaints to be appealed. 

Do we need to say anything about the situation where an IAB member is involved in the dispute? Or the entire IAB?

Reply:
How about - 
```suggestion
any SAA decision could be appealed to the IAB. The IAB 
shall then review the situation and attempt to resolve it in 
a manner of its own choosing.
```

Do we need to say anything more?